src/components/snackbar/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { enqueueSnackbar } from 'notistack';
+
+// type Props = {
+//   messageError: string;
+//   messageSuccess: string;
+// };
+
+const enqueueSnackbarError = (message: string) => {
+  return enqueueSnackbar(message, {
+    variant: 'error',
+    anchorOrigin: {
+      vertical: 'top',
+      horizontal: 'right',
+    },
+  });
+};
+
+const enqueueSnackbarSuccess = (message: string) => {
+  return enqueueSnackbar(message, {
+    variant: 'success',
+    anchorOrigin: {
+      vertical: 'top',
+      horizontal: 'right',
+    },
+  });
+};
+
+export const useSnackBars = () => {
+  // const { messageError, messageSuccess } = props;
+
+  const error = enqueueSnackbarError;
+  const success = enqueueSnackbarSuccess;
+
+  return { error, success };
+};
```

Comment:
Dziwne, że nie trzeba użyć hooka `useSnackbar();`, ale dla pewności i patrząc na dokumentację możemy zmienić to na:

```suggestion
import { useSnackbar, OptionsObject } from 'notistack';

export const useSnackbars = () => {
  const { enqueueSnackbar } = useSnackbar();
  
  const enqueueSnackbarError = (message: string) => {
    return enqueueSnackbar(message, {
      variant: 'error',
      ...sharedOptions
    });
  };

  const enqueueSnackbarSuccess = (message: string) => {
    return enqueueSnackbar(message, {
      variant: 'success',
      ...sharedOptions
    });
  };

  return { enqueueSnackbarError, enqueueSnackbarSuccess };
};

const sharedOptions: OptionsObject = {
  anchorOrigin: {
    vertical: "top",
    horizontal: "right",
  },
};
```